certbot/certbot/compat/os.py

```diff
@@ -3,8 +3,11 @@
 (e.g. chown, chmod, getuid) that would be harmful to the Windows file security model of Certbot.
 This module is intended to replace standard os module throughout certbot projects (except acme).
 
-isort:skip_file
+This module has the same API as the os module in the Python standard library
+except for the functions defined below.
+
 """
+# isort:skip_file
```

Comment:
why is this now commented?

Reply:
Because this string is currently shown as documentation at https://certbot.eff.org/docs/api/certbot.compat.os.html.

I tested changing this to a comment with our currently pinned version of `isort` and it still properly skips the file.

Reply:
huh, cool.